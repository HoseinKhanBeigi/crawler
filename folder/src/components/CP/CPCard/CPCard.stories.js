import React from 'react';
import { CPCardTest } from './CPCard';

export default {
  component: CPCardTest,
  title: 'Form/CPCard',
};

export const CPCard = () => <CPCardTest className="classname">ورود</CPCardTest>;
