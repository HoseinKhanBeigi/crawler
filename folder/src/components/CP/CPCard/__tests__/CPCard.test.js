/* eslint-env jest */
/* eslint-disable padded-blocks, no-unused-expressions */

import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { CPCardTest as CPCard } from '../CPCard';

configure({ adapter: new Adapter() });

describe('Header', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<CPCard title="Events" />);
  });

  it('should render children when passed in', () => {
    expect(wrapper.prop('title')).toEqual('Events');
  });

  test('should render children when passed in', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
