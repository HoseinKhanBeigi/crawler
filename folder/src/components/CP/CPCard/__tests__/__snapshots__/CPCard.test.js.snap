// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render children when passed in 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <CPCard
    actions={Array []}
    className={null}
    close={false}
    cover=""
    metaAvatar={null}
    metaDesc=""
    metaTitle=""
    more={null}
    noBorder={false}
    title="Events"
    toggle={false}
    type="default"
  >
    
  </CPCard>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "actions": Array [],
      "bordered": true,
      "children": "",
      "className": "card default",
      "cover": "",
      "extra": null,
      "title": "Events",
    },
    "ref": null,
    "rendered": "",
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "actions": Array [],
        "bordered": true,
        "children": "",
        "className": "card default",
        "cover": "",
        "extra": null,
        "title": "Events",
      },
      "ref": null,
      "rendered": "",
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
